docs/BR.md

```diff
@@ -751,7 +751,14 @@ The Random Value SHALL remain valid for use in a confirming response for no more
 
 **Note**: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the Domain Labels of the validated FQDN. This method is suitable for validating Wildcard Domain Names.
 
-Effective November 1, 2024, Subscriber Certificates MUST NOT be issued that 1) rely on WHOIS to identify Domain Contact information or 2) reuse validation data where WHOIS was used to identify the Domain Contact.  
+Effective January 15, 2025, when issuing Subscriber Certificates…
+- The CA MUST NOT rely on Domain Contact information obtained using an HTTPS website, regardless of whether previously obtained information is within the allowed reuse period.
+- The CA MUST NOT rely on Domain Contact information obtained using the WHOIS protocol (RFC 3912) or the Registry Data Access Protocol (RFC 7482) if the requested Domain Name contains a ccTLD, regardless of whether previously obtained information is within the allowed reuse period.
+- When obtaining Domain Contact information using the WHOIS protocol, the CA MUST query IANA's WHOIS server and follow referrals to the appropriate gTLD WHOIS server.
+- When obtaining Domain Contact information using the Registry Data Access Protocol, the CA MUST utilize IANA's bootstrap file to identify and query the correct RDAP server for the domain.
+- The CA SHOULD NOT rely on cached 1) WHOIS server information or 2) RDAP bootstrap data from IANA to ensure that it relies upon up-to-date and accurate information.
+
+Effective September 15, 2025, the CA MUST NOT rely on this method.
```

Comment:
We should also specify what happens with data reuse, i.e. copy this from 3.2.2.4.3: "Prior validations using this method and validation data gathered according to this method SHALL NOT be used to issue certificates."

Reply:
@wthayer - thanks for your comment. To confirm my understanding, the proposal would be to similar to the following?

**From:** _Effective September 15, 2025, the CA MUST NOT rely on this method._

**To:** _Effective September 15, 2025:_
_- the CA MUST NOT rely on this method._
_- Prior validations using this method and validation data gathered according to this method MUST NOT be used to issue certificates._

[Pedantic] Is there well-accepted convention within the Forum re: "MUST" vs "SHALL"? Generally speaking, I'm on Team "[MUST](https://www.plainlanguage.gov/guidelines/conversational/shall-and-must/#:~:text=Use%20%E2%80%9Cmust%E2%80%9D%20not%20%E2%80%9Cshall,express%20a%20requirement%20or%20obligation.)."

Reply:
Yes, that's my suggestion, thank you.

I'm not an expert in IETF-speak, but I think it's fine to use MUST here. I simply copied the SHALL language from a different subsection.

Reply:
Given no further comments, I actioned this comment in [b740449](https://github.com/ryancdickson/staging/commit/b740449d80c71ec4f3cefab2e68eb86ba2ccc924).